Keda.CosmosDbScaler/Protos/externalscaler.proto

```diff
@@ -1,51 +1,45 @@
-syntax = "proto3";
-
-option csharp_namespace = "Keda.CosmosDB.Scaler.Protos";
-
-package externalscaler;
-option go_package = ".;externalscaler";
-
-service ExternalScaler {
-    rpc IsActive(ScaledObjectRef) returns (IsActiveResponse) {}
-    rpc StreamIsActive(ScaledObjectRef) returns (stream IsActiveResponse) {}
-    rpc GetMetricSpec(ScaledObjectRef) returns (GetMetricSpecResponse) {}
-    rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {}
-}
-
-message ScaledObjectRef {
-    string name = 1;
-    string namespace = 2;
-    map<string, string> scalerMetadata = 3;
-}
-
-message IsActiveResponse {
-    bool result = 1;
-}
-
-message GetMetricSpecResponse {
-    repeated MetricSpec metricSpecs = 1;
-}
-
-message MetricSpec {
-    string metricName = 1;
-    int64 targetSize = 2;
-}
-
-message GetMetricsRequest {
-    ScaledObjectRef scaledObjectRef = 1;
-    string metricName = 2;
-}
-
-message GetMetricsResponse {
-    repeated MetricValue metricValues = 1;
-}
-
-message MetricValue {
-    string metricName = 1;
-    int64 metricValue = 2;
-}
-
-message NewRequest {
-	ScaledObjectRef scaledObjectRef = 1;
-	map<string, string> metadata = 2;
+syntax = "proto3";
+
+option csharp_namespace = "Keda.CosmosDbScaler";
```

Comment:
any strong reason to update the naming convention from `CosmosDb.Scaler` to `CosmosDBScaler` ?

Reply:
The namespace-tree was shortened to keep the count of namespace-branches in the test of demo projects smaller. Since the namespace gets reused for project-file name and assembly name, this helps reducing extra dots in the filenames. If we need to use three-node namespace, I would prefer `Keda.Scalers.CosmosDb` instead of `Keda.CosmosDb.Scaler` as we won't have any other namespace with `Keda.CosmosDb.` as prefix, but we can have `Keda.Scalers.<event-source>` as namespace in a different `kedacore` repo.